scripts/check-deps.sh

```diff
@@ -28,9 +28,6 @@ for dep in $(LC_ALL=C sort -u "${tmpdir}/golist"); do
 		# ignore ourselves
 		github.com/influxdata/telegraf) continue;;
 
-		# dependency is replaced in go.mod
```

Comment:
Related to https://github.com/satori/go.uuid/issues/73, is this resolved?

Reply:
> Were you able to run the integration tests? As we don't currently run them as part of our circleci checks.

I ran the appropriate integration tests locally and looking especially at the tests where I made changes to the actual plugins.

Reply:
The problem should have been fixed [here](https://github.com/satori/go.uuid/pull/75). However when I looked at the repository a second time, I noticed that it's not really maintained. For this reason, I think it is better to implement the replace function.